taxi/models.py

```diff
@@ -15,7 +16,19 @@ def __str__(self):
 
 
 class Driver(AbstractUser):
-    license_number = models.CharField(max_length=255, unique=True)
+    license_number = models.CharField(
+        max_length=255,
+        unique=True,
+        validators=[
+            RegexValidator(
+                regex=r"^[A-Z]{3}\d{5}$",
+                message="Number of license must consist "
+                        "only of 8 characters, first 3 characters "
+                        "are uppercase letters, "
+                        "last 5 characters are digits."
```

Comment:
The regex pattern `r"^[A-Z]{3}\d{5}$"` requires exactly 3 uppercase letters followed by 5 digits, which matches the message description. Ensure that this pattern aligns with the actual requirements for a license number. If the requirements differ, adjust the regex pattern or the message accordingly.